runelite-client/src/main/java/net/runelite/client/plugins/corp/CorpPlugin.java

```diff
@@ -202,4 +206,48 @@ public void onInteractingChanged(InteractingChanged interactingChanged)
 
 		players.add(source);
 	}
+
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		if (client.getGameState() != GameState.LOGGED_IN)
+		{
+			return;
+		}
+		String option = Text.removeTags(event.getOption()).toLowerCase();
+		String target = Text.removeTags(event.getTarget()).toLowerCase();
+		if (option.equals("attack"))
+		{
+			if (config.leftClickableCore() && target.contains("dark energy core"))
```

Comment:
I would put this config check with the gamestate check, to prevent from even running anything at all when disabled.